packages/core/database/lib/entity-manager/index.js

```diff
@@ -1164,6 +1170,50 @@ const createEntityManager = (db) => {
       }
     },
 
+    // TODO: Clone polymorphic relations
+    // TODO: Excluded relation attributes
+    /**
+     *
+     * @param {*} uid - uid of the entity to clone
+     * @param {*} id - id of the entity to clone
+     * @param {*} cloneId - id of the cloned entity
+     * @param {object} opt
+     * @param {object} opt.cloneAttrs - key value pair of attributes to clone
+     * @param {object} opt.transaction - transaction to use
+     * @example cloneRelations('article', 1, 2, { cloneAttrs: { categories: true } })
+     */
+    async cloneRelations(uid, id, cloneId, { cloneAttrs = {}, transaction }) {
+      const { attributes } = db.metadata.get(uid);
+
+      if (!attributes) {
+        return;
+      }
+
+      await mapAsync(Object.entries(cloneAttrs), async ([attrName, shouldClone]) => {
+        if (!shouldClone) return;
+        const attribute = attributes[attrName];
+
+        if (attribute.type !== 'relation') {
+          throw new DatabaseError(
+            `Attribute ${attrName} is not a relation attribute. Cloning relations is only supported for relation attributes.`
+          );
+        }
+
+        // TODO: add support for cloning polymorphic relations
+        if (!attribute.joinTable) {
```

Comment:
What about inline relations that use joinColumn instead of joinTable?

Reply:
Basically, won't this remove the possibility to clone createdBy & updatedBy on regular content-types?

Reply:
> for one to one join columns, (non bidirectional) it should be handled automatically by cloning the entity (which will be implemented in a follow up PR).

Makes sense :slightly_smiling_face: 

> For the rest of relations, we could prepare an extension for them, but they are not inside the scope of the current feature. Do you think they should be prioritised too?

If not prioritized, maybe just adding a special condition for them with a todo could make it clear it's something that should be implemented at some point (in case someone who's not aware of those impl details read this code)

Reply:
makes sense! will do that. 
And ty for the review!!